### ISSUE TITLE
How to install R on JupyterLab ? 

### ISSUE THREAD

**Issue by jbastruz** [YunoHost-Apps/jupyterlab_ynh#114]
Hi everyone :)

I'm looking for a way to install RKernel on JupyterLab. the point is every method I read didn't work (mainly because no condo nor jupyter function are installed on my server (but I have installed Jupyter !))

if someone has a solution to my issue, i'd love to ear the answer :)

Thanks

**Comment by agdamsbo** [comment]
@jbastruz, did you solve this?

**Comment by agdamsbo** [comment]
So, I found a solution, and will just leave it here for reference:

I forked the `IRkernel` and modified the `installspec()` to allow to specify the jupyter path. After installing R and the IRkernel package, then run the following:
```
source("https://github.com/agdamsbo/IRkernel/blob/master/R/installspec.r")
installspec(user=FALSE, jupyter_path="/opt/jupyterlab/venv/bin/jupyter")
```
Test steps in the function assumes the location of jupyter, which is not correct when installing jupyter through YunoHost.
After this, jupyterlab should recognise the R kernel.
